Source/BusinessLogic/Logic/Points/AverageGameDurationTierCalculator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using BusinessLogic.Models.PlayedGames;
+
```

Comment:
@jakejgordon Conditionals like this for me are a smell, not saying its a bad smell, I'm just commenting because I'd like to see how this is used to get more context :)